### PR TITLE
Update to ABP v5.1.0

### PR DESCRIPTION
**Pull request by acjh**

### CODE REVIEW

File: Abp/Framework/scripts/abp.d.ts [hunk 1]
```diff
@@ -111,19 +111,6 @@
...
-        /**
-         * Saves refreshToken token.
-         * @param refreshToken The token to be saved.
-         * @param expireDate Optional expire date. If not specified, token will be deleted at end of the session.
-         */
-        function setRefreshToken(refreshToken: string, expireDate?: Date): void;
-
-        function getRefreshToken(): string;
-
-        function clearRefreshToken(): void;
```
Comment (acjh) [inline Abp/Framework/scripts/abp.d.ts]: Need to restore this (added directly by #21) and add to ABP repo.

**Comment by acjh** [comment]
@hikalkan @ismcagdas Please help to publish abp-web-resources 5.1.0 to npm.

**Comment by ismcagdas** [comment]
Done https://www.npmjs.com/package/abp-web-resources

**Comment by acjh** [comment]
Oh, actually I incremented the version in this PR to sync it with **bower.json**.

**Comment by ismcagdas** [comment]
@acjh oh sorry, I haven't seen that.

**Pull request merged**